content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -193,6 +193,100 @@ it does the following when a Pod is created:
 1. If the spec of the incoming Pod doesn't already contain any `imagePullSecrets`, then the
    admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
 
+### Legacy ServiceAccount token cleaner
+
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+Starting with Kubernetes version 1.28, a new feature called
```

Comment:
This feature is already GA'ed. Just mention that in a different section:

### Legacy ServiceAccount token tracking controller

This controller generates a ConfigMap `kube-system/kube-apiserver-legacy-service-account-token-tracking` to record the timestamp when legacy service account tokens began to be monitored by the system.

Reply:
Done.